.github/workflows/reset-pr-branch.yml

```diff
@@ -0,0 +1,66 @@
+---
+name: Reset PR Branch
+
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+concurrency:
+  group: argocd-reset-pr-${{ github.event.issue.number }}
+  cancel-in-progress: false
+
+jobs:
+  reset-pr:
+    if: |
+      contains(github.event.comment.body, '/reset') &&
```

Comment:
We should require this and /deploy to be at the beginning of the comment, not just anywhere and be the only thing in the comment, other than whitespace.